### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by andrewkozma** [woshibobo/ORIE-4741-Project#10]
I liked reading this report and I liked the question you were trying to ask. I thought it was very relevant to the class: you are given a big, messy dataset, and you must use different techniques to create a linear model in order to predict housing prices.

First off, I thought it was very cool how you could use the feature transformations you did to convert ordinal and nominal values into vectors. This became very useful because you were able to use them in your model. I liked the first chart you had that showed the correlation between all the variables. I liked how it enabled me to visualize all the features you had and how they were related. I liked too how you tried the different loss functions and regularizers we went over in class. 

It concerned me though, like you mentioned in the report, that you had didn't have that many examples in your data. This, coupled with the large number of features present after the feature transformations, could have led to overfitting. Another thing that I wish was present in the report was further analysis. There was some excellent loss functions and regularizers going on, but I didn't see any mention of the most significant and important features in determining housing prices. The main focus of the conclusion was rather that this loss function and regularizer combination produced the lowest mean squared error. I would have liked to read about these significant factors in your report. 

I liked how you mentioned that the next thing you were going to consider in your dataset was to run PCA. I thought that would have been an interesting extension of your analysis and could help you to further discover the most important features in the price of a house.
